code/03_census_and_bed_days.R

```diff
@@ -0,0 +1,868 @@
+  ungroup()
+
+# datafile5a<-datafile2
+# 
+# datafile5a$level<-"1"
+# datafile5a$areaname<-"Scotland"
+# datafile5a$age_grp<-"All"
+# datafile5a$reason_grp_high_level<-"All"
+# 
+# #copy secondary Code 9 codes into delay reason - e.g. delay reason 9 becomes 51X etc..
+# #then update reas1 with individual delay codes in delayreason
+# 
+# datafile5a<-datafile5a %>% mutate(REASONFORDELAY=
+#                                   if_else(REASONFORDELAY==9,dd_code_2,REASONFORDELAY))
+# 
+# ScotlandindreasonsALL<- datafile5a %>% 
+#   group_by(year,monthflag,level,areaname,age_grp,reason_grp_high_level) %>% 
+#   summarise(num_pats=sum(num_pats,na.rm=TRUE),delay_1_to_3_days=sum(delay_1_to_3_days,na.rm=TRUE),
+#             delay_3_to_14_days=sum(delay_3_to_14_days,na.rm=TRUE),delay_2_to_4_weeks=sum(delay_2_to_4_weeks,na.rm=TRUE),
+#             delay_4_to_6_weeks=sum(delay_4_to_6_weeks,na.rm=TRUE),delay_6_to_12_weeks=sum(delay_6_to_12_weeks,na.rm=TRUE),
+#             delay_3_to_6_months=sum(delay_3_to_6_months,na.rm=TRUE),delay_6_to_12_months=sum(delay_6_to_12_months,na.rm=TRUE),
+#             delay_over_12_months=sum(delay_over_12_months,na.rm=TRUE),delay_over_3_days=sum(delay_over_3_days,na.rm=TRUE),
+#             delay_under_2_weeks=sum(delay_under_2_weeks,na.rm=TRUE),
+#             delay_over_6_weeks=sum(delay_over_6_weeks,na.rm=TRUE),delay_over_4_weeks=sum(delay_over_4_weeks,na.rm=TRUE),
+#             delay_over_2_weeks=sum(delay_over_2_weeks,na.rm=TRUE)) %>% 
+#   ungroup()
+
+#match all Scotland reasons and high level reason groupings back to main file and then add on individual reason breakdown
+
+
+#dataset6test <- rbind(Scotlandhighlevelgrouping, Tabs136, c(year, monthflag, level, areaname, age_grp, reason_grp_high_level))
+
+
+#datafile6 <- full_join(Tabs136,Scotlandhighlevelgrouping,
+#                         by = c(("year" = "year"), ("monthflag"="monthflag"),("level"="level"),("areaname"="areaname"), ("age_grp"="age_grp"),("reason_grp_high_level"="reason_grp_high_level")))
+
+#datafile6a <- full_join(datafile6,Scotlandallreasons,
+#                        by = c(("year" = "year"), ("monthflag"="monthflag"),("level"="level"),("areaname"="areaname"), ("age_grp"="age_grp"),("reason_grp_high_level"="reason_grp_high_level")))
+
+#datafile6b <- full_join(datafile6a,Scotlandsublevelgroupings,
+#                         by = c(("year" = "year"), ("monthflag"="monthflag"),("level"="level"),("areaname"="areaname"), ("age_grp"="age_grp"),("reason_grp_high_level"="reason_grp_high_level")))
+
+
+#match files ( excluding files with mean and media - no longer needed )
+
+
+datafile6<- bind_rows(Tabs136,Scotlandindreasons)
+
+#datafile6 <- left_join(Tabs136, Scotlandindreasons,
+#                       by = c(("year" = "year"), ("monthflag"="monthflag"),("level"="level"),("areaname"="areaname"), ("age_grp"="age_grp"),("reason_grp_high_level"="reason_grp_high_level")))
+
+
+
+datafile7<- bind_rows(datafile6,Scotlandindreasons)
```

Comment:
See "Style Guide" and "Incremental Data-Frame Names" sections in general feedback. This is actually creating a new data-frame, however, so maybe using a more descriptive name would be useful?
```suggestion
datafile7 <- bind_rows(datafile6, Scotlandindreasons)
```